lamindb/_ulabel.py

```diff
@@ -31,9 +32,8 @@ def __init__(self, *args, **kwargs):
         kwargs.pop("reference_type") if "reference_type" in kwargs else None
     )
     if len(kwargs) > 0:
-        raise ValueError(
-            "Only name, description, reference, reference_type are valid keyword arguments"
-        )
+        valid_keywords = ", ".join([val[0] for val in _get_record_kwargs(ULabel)])
+        raise ValueError(f"Only {valid_keywords} are valid keyword arguments")
```

Comment:
Why is this a ValueError and not a FieldValidationError or ValidationError?

Reply:
Now all FieldValidationError. You were looking at my PR before I rerequested a review ^_^